app/javascript/flavours/glitch/components/edited_timestamp/index.jsx

```diff
@@ -68,3 +66,5 @@ class EditedTimestamp extends React.PureComponent {
   }
 
 }
+
+export default connect(null, mapDispatchToProps)(EditedTimestamp);
```

Comment:
```suggestion
export default connect(null, mapDispatchToProps)(injectIntl(EditedTimestamp));
```
Missing `injectIntl`